buildkite/pipelines/bacalhau-golang.yaml

```diff
@@ -64,9 +63,14 @@ steps:
 
   - wait: ~
 
-  # Release CLI is only triggered if the tag is present.
+  - label: ":docker: Build & Publish Bacalhau Image"
+    command: "./buildkite/scripts/bacalhau_image.sh"
+    key: "publish-bacalhau-image"
+    agents:
+      queue: "buildkite-hosted-linux-large"
+    if: build.pull_request.branch =~ /ud/
```

Comment:
should this be main? actually do we care about the branch name if the build is triggered by a release? There care be scenarios where we release and or create a tag not from main

Reply:
Yes this was for testing. Reverting it !